tr/cs-221-states-models.md

```diff
@@ -0,0 +1,980 @@
+
+&#10230; Algoritma ― A∗, s durumu ile send bitiş durumu arasındaki en kısa yolu bulmayı amaçlayan bir arama algoritmasıdır. Bahse konu algoritma PastCost(s)+h(s)'yi artan sıra ile araştırır. Aşağıda verilenler ışığında kenar maliyetlerini de içeren tek tip maliyet aramasına eşittir:
+
+<br>
+
+
+**43. Remark: this algorithm can be seen as a biased version of UCS exploring states estimated to be closer to the end state.**
+
+&#10230; Not: Bu algoritma, son duruma yakın olduğu tahmin edilen durumları araştıran tek tip maliyet aramasının taraflı bir sürümü olarak görülebilir.
+
+<br>
+
+
+**44. [Consistency ― A heuristic h is said to be consistent if it satisfies the two following properties:, For all states s and actions a, The end state verifies the following:]**
+
+&#10230; [Tutarlılık ― Bir sezgisel h, aşağıdaki iki özelliği sağlaması durumunda tutarlıdır denilebilir:, Bütün s durumları ve a eylemleri için, bitiş durumu aşağıdakileri doğrular:]
+
+<br>
+
+
+**45. Correctness ― If h is consistent, then A∗ returns the minimum cost path.**
+
+&#10230; Doğruluk ― Eğer h tutarlı ise o zaman A∗ algoritması asgari (minimum) maliyet yolunu döndürür.
+
+<br>
+
+
+**46. Admissibility ― A heuristic h is said to be admissible if we have:**
+
+&#10230; Kabul edilebilirlik ― Bir sezgisel h kabul edilebilirdir eğer:
+
+<br>
+
+
+**47. Theorem ― Let h(s) be a given heuristic. We have:**
+
+&#10230; Teorem ― h(s) verilen sezgisel olsun ve:
+
+<br>
+
+
+**48. [consistent, admissible]**
+
+&#10230; [tutarlı, kabul edilebilir]
+
+<br>
+
+
+**49. Efficiency ― A* explores all states s satisfying the following equation:**
+
+&#10230; Verimlilik ― A* algoritması aşağıdaki eşitliği sağlayan bütün s durumlarını araştırır:
+
+<br>
+
+
+**50. Remark: larger values of h(s) is better as this equation shows it will restrict the set of states s going to be explored.**
+
+&#10230; Not: h(s)'nin yüksek değerleri, bu eşitliğin araştırılacak olan s durum kümesini kısıtlayacak olması nedeniyle daha iyidir.
+
+<br>
+
+
+**51. Relaxation**
+
+&#10230; Rahatlama
+
+<br>
+
+
+**52. It is a framework for producing consistent heuristics. The idea is to find closed-form reduced costs by removing constraints and use them as heuristics.**
+
+&#10230; Bu tutarlı sezgisel için bir altyapıdır (framework). Buradaki fikir, kısıtlamaları kaldırarak kapalı şekilli (closed-form) düşük maliyetler bulmak ve bunları sezgisel olarak kullanmaktır.
+
+<br>
+
+
+**53. Relaxed search problem ― The relaxation of search problem P with costs Cost is noted Prel with costs Costrel, and satisfies the identity:**
+
+&#10230; Rahat arama problemi ― Cost maliyetli bir arama probleminin rahatlaması, Costrel maliyetli Prel ile ifade edilir ve kimliği (satisfy) karşılar:
```

Comment:
Rahat arama problemi (Relaxed search problem) ― Cost maliyetli bir arama probleminin rahatlaması, Costrel maliyetli Prel ile ifade edilir ve kimliği karşılar (satisfies the identity) :

Reply:
OK